lab_6_pipeline/pipeline.py

```diff
@@ -112,11 +198,37 @@ def convert_morphological_tags(self, tags: str) -> str:  # type: ignore
         """
         Converts the Mystem tags into the UD format
         """
+        part_of_speech = self.convert_pos(re.findall(r'[A-Z]+', tags)[0])
+        gramm_categories = {'NOUN': [self.gender, self.animacy, self.case, self.number],
```

Comment:
great! 🎖️ (but again formatting)
```suggestion
        gramm_categories = {
              'NOUN': [self.gender, self.animacy, self.case, self.number],
```